CMakeLists.txt

```diff
@@ -122,13 +99,28 @@ set(NACL_VM_INHERITED_OPTIONS ${NACL_VM_INHERITED_OPTIONS} RC_MAX_LAYERS RC_MAX_
 
 if (BUILDING_ANY_GAMELOGIC)
     if (BUILD_CGAME)
-        # Freetype
+        # Freetype (RmlUi dependency), already found for a native game when the client is built.
```

Comment:
Does something bad happen if you do `prefer_package` again for one that was searched earlier? Would be nicer not to care about that

Reply:
Not sure to fully understand what you mean, but replacing some existing stuff with `prefer_package` should not have more drawbacks than replacing the same existing stuff by something else written by hand. If there is a drawback, then we would have to handle it anyway.

But I doubt that using a submodule after a search is problematic, I made efforts for the submodules using the same variable names as the search, so one should fully replace the other. I haven't got any problem by turning on and off the `PREFER_EXTERNAL_LIBS` option. And if one day we face a cmake error, that would be one we cannot avoid anyway (i.e. a CMake bug or design limitation).

I don't see anything to do more.